packages/patternfly-4/react-integration/cypress/integration/breadcrumb.spec.ts

```diff
@@ -0,0 +1,7 @@
+describe('Breadcrumb Demo Test', () => {
```

Comment:
You need to verify something a the component level.  With this test, we don't even  know if the component loaded.